tests/flag_util_test.py

```diff
@@ -120,7 +120,11 @@ def testReadAndWrite(self):
     flag_values_copy.test_flag = 1
     self.assertFlagState(flag_values, 0, False)
     self.assertFlagState(flag_values_copy, 1, False)
-    with flag_util.FlagDictSubstitution(flag_values, flag_values_copy.FlagDict):
+    if hasattr(flag_values_copy, '_flags'):
+      flag_dict_func = flag_values_copy._flags
+    else:
+      flag_dict_func = flag_values_copy.FlagDict
```

Comment:
FlagDict is going away internally, and unfortunately _flags is not available in github.com/google/python-gflags (yet), and the test needs to run in both environments.

I'm going to bring the two flags library to a same state, but it takes sometime, that results in this ugly intermediate state on your code :(